### DISCUSSION TITLE
GSoC 2023 Projects Announced

### DISCUSSION THREAD

**Discussion by mshudrak** [google/gcp_scanner#146] [Announcements]
Dear Community,

We are pleased to announce that our organization has been accepted to host 3 projects this year. We would like to thank all the applicants who submitted their proposals, contributed to GCP Scanner and took the time to participate in the application process. The GSoC 2023 should have already notified accepted candidates by email.

To those who have been selected, congratulations! You have been chosen for your excellent project proposals, prior contribution, relevant skills, and enthusiasm for open-source development. We are excited to have you on board and look forward to working with you on your projects this summer. We believe your contributions will greatly impact the open-source community. We will individually contact each of the selected candidates with further instructions shortly.

For those who have not been selected, please do not be discouraged. We had more than 60 proposals, and unfortunately, we could only select a limited number of them. We appreciate the time and effort you put into your proposals, and we encourage you to keep up the good work. We hope that you will continue to pursue your interests in open-source development and that you will apply again in the future.

Again, congratulations to all selected candidates. We look forward to an exciting and productive summer! 

Sincerely,

Maksim Shudrak on behalf of GCP Scanner Mentors Team

**Comment by sudiptob2** [comment]
My proposal, **Expanding Support for GCP Resources with Modularized Crawler** has been accepted for Google Summer of Code 2023 with the GCP Scanner organization✌️ I couldn't have achieved this without the support and guidance of the community, and I'm truly grateful for the opportunity to contribute to this fantastic organization 💯
